### ISSUE TITLE
Buttons used within videos do not have screenreader-compatible aria-label attributes for accessibility compliance

### ISSUE THREAD

**Issue by logan-reynolds** [kaltura/moodle_plugin#285]
The buttons used within these videos either do not have proper names or they do but have a redundant title attribute. Not all assistive devices support the title attribute so it is not a reliable tool to use for labeling purposes on the buttons in which this is the case. In regards to the other buttons, the title attribute should serve for advisory information and not be a copy of the button names.  Each of these buttons should be given a name using the aria-label attribute. The name can be the same as the one provided in the title. Then the title attribute should be removed from these buttons.

**Comment by muli** [comment]
Hi @logan-reynolds,

Can you please elaborate, what do you mean by "buttons used within videos"? Do you mean player buttons, like "full screen", "play", "pause", etc., or something else?

A screenshot will be helpful.

Thanks.

**Comment by ghost** [comment]
Yes, the video control buttons are the issue. Each of these buttons should be given a name using the aria-label attribute. The name can be the same as the one provided in the title. Then the title attribute should be removed from these buttons.

**Comment by idodido** [comment]
@logan-reynolds make sure you are using the most recent Kaltura player. You can upgrade it in the KMC studio. Give it 5-15 min after the save as the player is cached.
